packages/eas-cli/src/credentials/ios/IosCredentialsProvider.ts

```diff
@@ -38,20 +32,23 @@ interface AppLookupParams {
 
 export default class IosCredentialsProvider implements CredentialsProvider {
   public readonly platform = Platform.IOS;
+  private readonly appLookupParams: GraphQLAppLookupParams;
 
-  constructor(private ctx: Context, private options: Options) {}
+  constructor(private ctx: Context, private options: Options) {
+    this.appLookupParams = getAppLookupParamsFromContext(this.ctx);
```

Comment:
options object already contains appLookupParams, it should be the same as the one in context, but values from context were not intended to be used this way.

The reason that values are available in context is to support a situation where you have actions that are not specific to that app, but might have an optional effect on a specific app e.g. create distribution certificate action should create dist cert that is not assigned to project, but we might prompt the user if they want to assign it to the current one. This use case is different, CredentialsProvider from the very beginning is running in context of specific project, so values should be explicitly passed.

I assume you did it because you need graphql app lookup params, but instead, I suggest just changing the constructor params type